README.md

```diff
@@ -19,6 +19,6 @@ Things you may want to cover:
 
 * Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+* Deployment instruction
```

Comment:
- [ ] [REQUIRED] Kindly ensure you have a professional README for this project. The README gives a new user/ visitor an idea of the project